### ISSUE TITLE
KBFS not usable after account reset

### ISSUE THREAD

**Issue by fd** [keybase/client#2472]
I reset my account. Now I can't access KBFS:

    /keybase/private/fd was last written by an unknown device claiming to belong to user fd. 
    The device has possibly been revoked by the user. 
    Use `keybase log send` to file an issue with the Keybase admins. [tags:FID=sARBmN7C8MdknRHpz_lrlg]

**Comment by cjb** [comment]
@fd Do you have a copy of your paper key?

**Comment by fd** [comment]
No, unfortunately 1password no longer stores password field histories... (just discovered that)

**Comment by strib** [comment]
@fd: can you please run `keybase log send` like the error recommended, and let us know the resulting ID?  It could be that you're running an old version of KBFS -- newer versions can recover from some instances of that errror.

And @cjb is asking about the your _paper key_, not your account passphrase.  It's the thing that the Keybase signup process asked you to put into your wallet.  In your case, for your old pre-reset account, the paper key was a string of words starting with "old race".  If you have that written down somewhere it may be possible to recover your data, in case you're already running the newest KBFS version.

**Comment by fd** [comment]
@strib No I lost my old paper key. Which means that I cannot recover my original KBFS (which is ok). I was just wondering if there is a way to reset my KBFS so that I can start using it again.

**Comment by strib** [comment]
Hi @fd, yes we can manually reset your KBFS public and private folders.  (We are working on an automatic way right now, but it's not ready yet.)

But we require a public, signed request to do so.  Please run the following command on one of your devices, and substitute the current date and time as indicated:

keybase sign -m "<DATE_AND_TIME>: Please completely reset folders /keybase/private/fd and /keybase/public/fd because I reset my Keybase account."

and post the output here. Thanks!

I would also still appreciate it if you could do a `keybase log send`, as that's the easiest way for us to figure out the internal IDs of the folders we need to reset it.  But if you'd rather not, we can figure it out other in another way.

**Comment by fd** [comment]
Here is my signed message (including `$(date)`)

```
BEGIN KEYBASE SALTPACK SIGNED MESSAGE. 
kXR7VktZdyH7rvq v5wcIkHbs0bDzao aviq0IDMH2cKvYq zKtbKOSKsZWqiap
WMEbUj6D9vnRYeY zv1CPmJEV0vj3vw 1ge46AY2xhtyybK 2TbKcKf13kjurnK 
wYzOrkPbj3jn1Xo j0NUOBhCWZB1P1l kLRZlQdVgC9Oz6v wRYvv6D9YJh0Cck 
XpeHe1OEwGIzjH0 pA9htDtTIMwdCF8 mwrAZJwpE12wcWu TaRN2PgjJi9yVcT 
HmYI9yAQpkmr8Uh RaqOI0sinHFxVgB a31ZFxvuJJ7cPer TkecL7soAGjrn6O 
GBFALhhW2kpUxZl CVqWlmWq9822xQ4 RzdFaeTv7BlOfDO 3a66x7ouerGIQoH 
95E0nTZRntuIjpu 5eOAQkslAU7uLNV JbIssvpwoEKFyPB bXzJ2R4yMh09T3y 
tJotEnke1T28u7K wlBHk9GY3zyADfA d3lpQIP56R3P2vm Ns.
END KEYBASE SALTPACK SIGNED MESSAGE.
```

And this is my log ID `fb5611fc6348ab136f29f01c`

**Comment by fd** [comment]
@strib Did you get around reseting my account already? I'm still unable to use KBFS.

**Comment by strib** [comment]
Thanks for the reminder @fd, we got distracted by a company retreat this week.  Will get this done today.

**Comment by strib** [comment]
@fd -- those folders should be reset now.  After you restart your local kbfs process, you should be good to go.  Let us know!

**Comment by nbolt** [comment]
I have the same issue. Have lost access to my kbfs and need everything reset. Has this process been automated yet?

**Comment by sean666888** [comment]
BEGIN KEYBASE SALTPACK SIGNED MESSAGE. kXR7VktZdyH7rvq v5wcIkHbsAlog7c gpTtNGIpaCtIeyN y5OiQnRPXscLpw5 gwNMuae1NUikqwr rxCmMn1D9BvcgHM IepAgrJzK3PXWC0 YgnJv3eFxAsX4lk DQQhBugXpL576Iw EkD7Ky79VzBxRWa eqJo1kMiyRIxY5F sUi973rbMzpNeKG JtVEdOM3w1T6iK6 wTnwvzWl3ef8s0R 8OQ20JwpE12wdI0 0c9vek7BZt9yeGg aw08Skd80lHr7T3 wEyOI0sinHFxVgB a31ZFxvuJJ7cPer TkecL7t4vipzhgO cDKXtx3yjstiJge Csiy0luPVBdhraP DLrY8ZYHlMkOV6D KXZTfuvHQFIp4Yz NfgwDVQ5x2lY3ab jDdWu5j9FJ0yJzI 4jTYNQnmG8JkG8T go0ZHs60guGhPXV Q9Y7K4NTa4Q6qX1 xoBlo3TLlg4APPo OL4FpG99L0ngLI2 ZY856IdgD3I. END KEYBASE SALTPACK SIGNED MESSAGE.
